CMakeLists.txt

```diff
@@ -140,7 +153,7 @@ if(NOT SKIP_LIBRARY)
   if( BUILD_WITH_TENSILE )
     # we will have expanded "all" for tensile to ensure consistency as we have local rules
     set( Tensile_ARCHITECTURE "${AMDGPU_TARGETS}" CACHE STRING "Tensile to use which architecture?" FORCE)
-    set( TENSILE_VERSION 4.38.0 CACHE STRING "The version of Tensile to be used")
+    set( TENSILE_VERSION 4.39.0 CACHE STRING "The version of Tensile to be used")
 
```

Comment:
I think we should still try and get Tensile to change and bump versions for next release at start of cycle not at end.
Maybe we can do this early in 6.1 before any staging promotion.